dist/js/app.js

```diff
@@ -484,54 +501,49 @@ function populateLatest() {
       //document.getElementById("latest-commitref").innerHTML = releasesJson[0].name;
       //document.getElementById("latest-commitref").href = releasesJson[0].name;
 
-      // create an array of the details for each binary that is attached to a release
+      // create an array of the details for each asset that is attached to a release
       var assetArray = [];
       var assetCounter = 0;
       releasesJson[0].assets.forEach(function() {
         assetArray.push(releasesJson[0].assets[assetCounter]);
         assetCounter++;
       });
 
-      // build the download links section with these binaries
-      var linuxDlButton = document.getElementById("linux-dl-button");
-      var windowsDlButton = document.getElementById("windows-dl-button");
-      var macDlButton = document.getElementById("mac-dl-button");
-      var linuxPlatformBlock = document.getElementById("latest-linux");
-      var windowsPlatformBlock = document.getElementById("latest-windows");
-      var macPlatformBlock = document.getElementById("latest-mac");
-
+      // for each asset attached to this release, check if it's a valid binary, then add a download block for it...
       assetCounter2 = 0;
-      assetArray.forEach(function() {     // iterate through the binaries attached to this release
```

Comment:
Usually when you have loop counters like `assetCounter` and a `assetCounter2` it's a scoping or naming code smell....